website/source/docs/enterprise/index.html.md

```diff
@@ -9,24 +9,57 @@ description: |-
+
+See the [Autopilot Upgrade](https://www.nomadproject.io/guides/operations/autopilot.html#upgrade-migrations) subsection for a thorough overview.
+
+### Enhanced Read Scalability
+Explicitly mark server nodes as non-voters. Non-voters will receive the replication stream but will not take part in quorum (required by the leader before log entries can be committed). Adding explicit non-voters will scale reads and scheduling without impacting write latency.
+
+See the [Autopilot Read Scalability](https://www.nomadproject.io/guides/operations/autopilot.html#server-read-and-scheduling-scaling) subsection for a thorough overview.
+
+### Redundancy Zones
+Deploy more server nodes than availability zones. For example, in an environment with three availability zones one can run one voter and one non-voter in each availability zone, for a total of six servers. If an availability zone is completely lost, only one voter will be lost, so the cluster remains available. If a voter is lost in an availability zone, Nomad will promote the non-voter to voter automatically, putting the hot standby server into service quickly.
+
+See the [Autopilot Redundancy Zones](https://www.nomadproject.io/guides/operations/autopilot.html#redundancy-zones) subsection for a thorough overview.
+
+## Governance & Policy
+Governance & Policy is an add-on module that enables users to securely operate Nomad at scale across multiple teams through features such as Namespaces, Resource Quotas, Preemption, Sentinel Policies, and more.
 
+### Namespaces
+Enable multiple teams to safely use a shared multi-region deployment and reduce cluster fleet size.  In Nomad Enterprise, a shared cluster can be partitioned into namespaces which allows jobs and their associated objects to be isolated from each other and other users of the cluster.
 
+Namespaces enhance the usability of a shared cluster by isolating teams from the jobs of others, provide fine grain access control to jobs when coupled with ACLs, and can prevent bad actors from negatively impacting the whole cluster.
+
+See the [Namespaces Guide](https://www.nomadproject.io/docs/enterprise/namespaces/index.html) for a thorough overview.
+
+### Resource Quotas
+Limit resource consumption across teams or projects to reduce waste and align budgets.  In Nomad Enterprise, operators can define quota specifications and apply them to namespaces. When a quota is attached to a namespace, the jobs within the namespace may not consume more resources than the quota specification allows.
+
+This allows operators to partition a shared cluster and ensure that no single actor can consume the whole resources of the cluster. See the Resource Quotas Guide for more details.
```

Comment:
How about something like below instead? 
>This allows operators to partition a shared cluster amongst different teams or projects, and ensure distribution of cluster resources according to business priority.

Reply:
@preetapan Take a look at the latest commit, should address these directly!